content/docs/alerting/configuration.md

```diff
@@ -12,6 +12,9 @@ While the command-line flags configure immutable system parameters, the
 configuration file defines inhibition rules, notification routing and
 notification receivers.
 
+An visual routing tree editor and label set matcher is available
+[here](/webtools/alerting/routing-tree-editor).
```

Comment:
I'm generally not a fan of "here" links.

Maybe `The [visual editor]() can assist in building routing trees` or similar?